pkg/controller/nodeipam/ipam/multinetwork_cloud_cidr_allocator_test.go

```diff
@@ -56,18 +56,21 @@ func network(name, gkeNetworkParamsName string) *networkv1.Network {
 }
 
 func gkeNetworkParams(name, vpc, subnet string, secRangeNames []string) *networkv1alpha1.GKENetworkParamSet {
-	return &networkv1alpha1.GKENetworkParamSet{
```

Comment:
Please add a test case where there are no secondary ranges in the GKE network params

Reply:
It's already covered by this - https://github.com/kubernetes/cloud-provider-gcp/blob/master/pkg/controller/nodeipam/ipam/multinetwork_cloud_cidr_allocator_test.go#L146